TABTestKit/Classes/Contexts/InteractionContext.swift

```diff
@@ -30,14 +30,28 @@ public extension InteractionContext {
 		states.forEach { element.await(not: $0) }
 	}
 	
-	func scroll(_ element: Scrollable, _ direction: ElementAttributes.Direction, until otherElement: Element, is state: ElementAttributes.State, maxTries: Int = 10) {
-		var numberOfTries = 0
-		repeat {
-			guard !otherElement.determine(state, timeout: 1) else { return }
-			numberOfTries += 1
-			element.scroll(direction)
-		} while numberOfTries <= maxTries
-		XCTFail("Ran of out tries waiting for element to become \(state)")
+	func scroll(_ element: Scrollable, _ direction: ElementAttributes.Direction, until otherElement: Element, is states: ElementAttributes.State..., maxTries: Int = 10) {
```

Comment:
This now supports scrolling waiting for multiple states, rather than just one. The reason this is useful is because when scrolling collection views, whos cells are not all created at the same time, you might first need to scroll until the cell exists, and then scroll until it's visible (which might be at the same time, but not necessarily with collection views or tables)